packages/reporters/cli/src/Log.js

```diff
@@ -38,55 +57,83 @@ export function Log({event}: StringOrErrorLogProps) {
   throw new Error('Unknown log event type');
 }
 
-function InfoLog({event}: StringLogProps) {
-  return <Text>{event.message}</Text>;
+function Hints({hints}: {hints: Array<string>, ...}) {
+  return (
+    <div>
+      <div>Hints:</div>
```

Comment:
the heading is probably not needed? it's probably clear from context that the below hints are hints. :)

Reply:
Sure makes sense I’ll update accordingly :)